dataflow/tests/issue3447/Test.java

```diff
@@ -0,0 +1,12 @@
+// Test case for Issue 3447:
+// https://github.com/typetools/checker-framework/issues/3447
+
+public class Test {
```

Comment:
I would name the file and class Issue3447.

Reply:
It seems that I cannot change the file name or class name from `Test` to `Issue3447`. It is hardcoded at [LiveVariable.java#L23](https://github.com/typetools/checker-framework/blob/master/dataflow/src/test/java/livevar/LiveVariable.java#L23).

Reply:
Ah, ok.